dev/io.openliberty.security.oidcclientcore.internal/test/io/openliberty/security/oidcclientcore/token/auth/PrivateKeyJwtAuthMethodTest.java

```diff
@@ -240,4 +291,13 @@ private void verifyPrivateKeyJwt(String jwt) throws JoseException, IOException {
         assertNotNull("Expected JWT to include a jti claim, but it did not.", jsonPayload.get("jti"));
     }
 
+    private MockedStatic<CryptoUtils> setupMockFips140_3() {
+        MockedStatic<CryptoUtils> cryptoUtilsMock = Mockito.mockStatic(CryptoUtils.class);
+        cryptoUtilsMock.when(CryptoUtils::isFips140_3EnabledWithBetaGuard).thenReturn(true);
+        cryptoUtilsMock.when(CryptoUtils::getFipsLevel).thenReturn("140-3");
```

Comment:
I don't believe we need to mock anything after like 296. the `cryptoUtilsMock.when(CryptoUtils::isFips140_3EnabledWithBetaGuard).thenReturn(true);` should short circuit the underlying methods.